test/WebJobs.Script.Tests.Integration/Host/WebJobsScriptHostServiceTests.cs

```diff
@@ -51,12 +51,12 @@ public WebJobsScriptHostServiceTests()
             };
             var wrappedHealthMonitorOptions = new OptionsWrapper<HostHealthMonitorOptions>(_healthMonitorOptions);
 
-            _mockJobHostEnvironment = new Mock<IScriptJobHostEnvironment>(MockBehavior.Strict);
-            _mockJobHostEnvironment.Setup(p => p.Shutdown())
-                .Callback(() =>
-                {
-                    _shutdownCalled = true;
-                });
+            _mockApplicationLifetime = new Mock<IApplicationLifetime>(MockBehavior.Loose);
```

Comment:
why Loose?

Reply:
It's not necessary to have a setup for IApplicationLifetime since we're just relying on aspnet core's implementation and for the tests below, we just check for whether or not it's invoked right?

---
In reply to: [369886075](https://github.com/Azure/azure-functions-host/pull/5437#discussion_r369886075) [](ancestors = 369886075)

Reply:
ok